logger/test_logger.py

```diff
@@ -13,6 +13,8 @@
                          ]
                          )
 def test_request_service(capsys, input, expected_output):
+    logger.log_stream.truncate(0)
+    logger.log_stream.seek(0)
```

Comment:
The inclusion of `test_azure_client.py` breaks the logger unit test due to the order in which the unit tests are run. `test_azure_client.py` writes to `stderr` before `test_logger.py` is run, causing this unit test to read garbage from `stderr`. These 2 lines clear `stderr` before running the test.